ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/router-general.yml

```diff
@@ -4,7 +4,7 @@ router_general:
     ipv6: 2001:beef:cafe::1
   nexthop_fast_failover: true
   vrfs:
-    BLUE-C2:
+    - destination_vrf: BLUE-C2
```

Comment:
@tgodaA I would prefer to use `name` as the key, since we are in the list called `vrfs`, and there might be other keys not concerning route leaking, so `destination_vrf` does not make sense for that case. What do you think?

Reply:
Agreed. I didn't think that case.

Reply:
@Shivani-chourasiya please change the primary key under `vrfs` to `name`. Thank you!

Reply:
@tgodaA I reset your review since there are pending changes still to be made based on this comment!

Reply:
Updated!